Tensile/Common.py

```diff
@@ -678,7 +678,7 @@
     # in TransposeLDS=1 case, use wider load to fetch elements in summation dimension from LDS
     # helps optimizing instruction scheduling between MFMA and nonMFMA instructions
 
-    "LocalReadVectorWidth":      [ -1, 1, 2, 4, 8 ],
+    # "LocalReadVectorWidth":      [ -1, 1, 2, 4, 8 ], # NOTE param mostly covered by VW already; need more clarification 
```

Comment:
in transposeLDS =1 ;  VW of local read do not have to follow  VW/ GRVW
e.g  for some MT shapes GRVW might be x2 , but local Read can still use x4
       LRVW might use x2 due to lack of register resources while GRVW =4. 
having LRVW gives us flexibility to de-link GRVW and LRVW..

Reply:
Thanks for clarification. In that case, I could set GRVW=4 and VW=2 separately in the config yaml. Here is code snippet generated with hgemm TN, GRVW=2 VW=2 TransposeLDS=1:

```
/******************************************/
/* Unroll Loop 1/2 - Begin                */
/******************************************/

label_0010: // LoopCopy1 
/* iter 0 */

/* local read a */
ds_read_b64 v[vgprValuA_X1_I0+0:vgprValuA_X1_I0+0+1], v[vgprLocalReadAddrA] offset:8 // L -> Reg lro=4 swapByteOffset=0 ti=32 vIdx=0 rIdx=0 oIdx=0 buffer=1 iui=0
buffer_load_dword v[vgprG2LA+0], v[vgprGlobalReadOffsetA+0], s[sgprSrdA:sgprSrdA+3], 0, offen offset:0 // G -> Reg 0_0_0_0
buffer_load_dword v[vgprG2LB+0], v[vgprGlobalReadOffsetB+0], s[sgprSrdB:sgprSrdB+3], 0, offen offset:0 // G -> Reg 0_0_0_0
```
Maybe I missed something. But the way I see it, the combo GRVW/LRVW does not allow you more flexibility than the combo GRVW/VW already allows.

Reply:
From PR description:

> 7. fixed local read vector width mapping to wrong unit in internal representation

@ramjana The intent was to fix a minor case where LRVW4 incorrectly maps `ds_read_b128` in hgemm kernels (should be `ds_read_b64`) and I was in for a quick fix. In any case, we can always revisit this issue later.

Reply:
LRVW is only for local read(S) and expected to use when data layout in LDS is K coalescing dimension.  Like i said, user can use LRVW & innerUnroll variable to go wider local reads in transposeLDS case. I have not implemented the code yet. 

I would like you to put back the variable and merge..

Reply:
That's a genuine usage scenario and I won't (and didn't) argue. 

Like I said, it's completely feasible to do *exact same thing* with `VectorWidth` & `InnerUnroll` and maybe alongside with `GlobalReadVectorWidth` and `StoreVectorWidth`. That 3 parameters already allow you complete control over a) global load/local store b) local read and c) global store. I would expect one to extend these existing parameters with TransposeLDS case to fulfill the said use case

That being said, I have added back the `LocalReadVectorWidth` parameter in latest commit.